vault/resource_token.go

```diff
@@ -328,13 +328,23 @@ func tokenRead(d *schema.ResourceData, meta interface{}) error {
 		d.Set("encrypted_client_token", "")
 	}
 
-	issueTime, err := time.Parse(time.RFC3339Nano, resp.Data["issue_time"].(string))
+	issueTimeStr, ok := resp.Data["issue_time"].(string)
+	if !ok {
+		return fmt.Errorf("error issue_time is not a string, got %v", resp.Data["issue_time"])
```

Comment:
```suggestion
		return fmt.Errorf("error issue_time is not a string, got %T", resp.Data["issue_time"])
```
If we output the type here it'll be much easier for the caller to debug it.